src/containers/App/component.js

```diff
@@ -64,12 +46,13 @@ export default class App extends React.Component {
   }
 
   getChildContext () {
-    const { dispatch, currentUser, isMobile, location } = this.props
-    return {dispatch, currentUser, isMobile, location}
+    const { currentUser, isMobile, location } = this.props
+    const { dispatch } = this.context
```

Comment:
(It makes me very uneasy that non of our tests caught this issue... but the required PropType for dispatch will at least give us that warning once I put this right again.)